packages/lib/Cargo.toml

```diff
@@ -52,7 +52,7 @@ async-compression = { version = "0.4", features = [
 ] }
 hyper = { version = "0.14", features = ["full"] }
 hyper-tungstenite = { version = "0.10" }
-tokio-tungstenite = { version = "0.19" }
+tokio-tungstenite = { version = "0.19", features = ["rustls-tls-native-roots"] }
```

Comment:
I think it might be better to use the `rustls-tls-webpki-roots` feature here, it seems that that one shares more dependencies with other `rustls` things in our lockfile - https://github.com/filiptibell/lune/blob/main/Cargo.lock#L1752